slotfills/sections/group.jsx

```diff
@@ -75,44 +79,34 @@ const Group = ({
             >
               <PanelBody>
                 <PanelRow>
-                  <Button
-                    onClick={() => removeElement(index)}
-                  >
-                    x
-                  </Button>
-                  {Object.keys(children).map((itemKey) => {
-                    const child = children[itemKey];
-                    const fieldKey = uuidv4();
-                    const {
-                      field_class: fieldClass,
-                      attributes: {
-                        rows = null,
-                      } = {},
-                      label = '',
-                    } = child;
-                    return (
-                      <>
-                        {fieldClass === 'text' && rows === null ? (
-                          <TextField
-                            key={fieldKey}
+                  <div style={{ flexDirection: 'column' }}>
+                    <div style={{
+                      display: 'flex',
+                      width: '100%',
+                      clear: 'both',
+                      justifyContent: 'space-between',
+                    }}
+                    >
+                      <DragHandle />
+                      <Button
+                        onClick={() => removeElement(index)}
+                      >
+                        x
+                      </Button>
+                    </div>
+                    <div>
+                      {Object.keys(children).map((itemKey) => {
+                        const child = children[itemKey];
+                        return (
+                          <FieldRouter
```

Comment:
Here `<FieldRouter>` does need a key because there's a loop. If the order of the fields isn't going to change, then you can use `itemKey` for this purpose.